package.json

```diff
@@ -19,5 +19,6 @@
     "dependencies": {
     },
     "devDependencies": {
+        "htmlparser2": "^3.9.2"
```

Comment:
htmlparser2 must be installed by the postinstall script. Without that we get runtime errors on user machines since the module is required by our plugin.